src/System.Console/src/System/ConsoleModifiers.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace System
 {
+    [Serializable]
```

Comment:
nit: from a style point of view I would prefer flags on top of serializable, although I don't really care much.